pre_commit/languages/docker.py

```diff
@@ -20,18 +19,19 @@
 
 def _is_in_docker() -> bool:
     try:
-        with open('/proc/1/cgroup', 'rb') as f:
-            return b'docker' in f.read()
```

Comment:
reading this as binary is intentional -- this file does not have a specified encoding

Reply:
Reverted back to reading as binary (had to decode the container ID, though).